Runtime/Scripts/GltFast.cs

```diff
@@ -2195,10 +2195,16 @@ async Task
 
         ImageFormat GetImageFormatFromPath(string path) {
             if(path.EndsWith(".png",StringComparison.OrdinalIgnoreCase)) return ImageFormat.PNG;
-            if(path.EndsWith(".jpg",StringComparison.OrdinalIgnoreCase)
+            if(path.EndsWith(".jpg",StringComparison.OrdinalIgnoreCase) 
                 || path.EndsWith(".jpeg",StringComparison.OrdinalIgnoreCase)) return ImageFormat.Jpeg;
             if(path.EndsWith(".ktx",StringComparison.OrdinalIgnoreCase)
                 || path.EndsWith(".ktx2",StringComparison.OrdinalIgnoreCase)) return ImageFormat.KTX;
+            if(path.IndexOf(".png", StringComparison.OrdinalIgnoreCase) >= 0) return ImageFormat.PNG;
```

Comment:
This is somewhat redundant to the conditions above and has a potential of creating false positives.
A better approach would be to search for the last occurance of `.`, assume it separates the extension and look at the 3-4 following chars.
Have a look at `UriHelper.IsGltfBinary` for reference. It does the same for .gltf/.glb

Reply:
I thought of something like that but query strings can contain periods so that would still not resolve the issue.
The redundancy is another matter and will fix. Was a perhaps misguided attempt to have some holistic performance, since the majority of gltfs won't have query strings.

Reply:
Since this is a specific fix to handle URL queries we could look from first "?" and forward to "." to find the file ending and compare that if you are worried about false positives for filenames or query strings with .png in them.

Reply:
That sounds good to me!